### PR TITLE
API phpunit 9 support

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10019

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,53 +1,4 @@
...
-
-script:
-  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit; fi
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml; fi
-  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs src/ tests/; fi
-
-after_success:
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then bash <(curl -s https://codecov.io/bash) -f coverage.xml; fi
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/standard-jobs-range.yml
```
Comment (maxime-rainville) [inline .travis.yml]: What's the point of using the travis CI build if travis is not running?

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Builds are failing and I don't have permission to re-run them.

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml]: Not alot :-) Removed

**Comment by emteknetnz** [comment]
@maxime-rainville failing unit test requires tagging a new minor of queuedjobs which requires unreleased framework ^4.10 - we should probably just merge this as is

**Comment by maxime-rainville** [comment]
I don't think tagging a new minor would be terrible ... composer wouldn't be incline to install it if it's set to prefer stable.

That's another example why I think not getting the manifest to ignore tests is not a viable choice. I'll go through the other PRs and come back to this one later on

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**